### PR TITLE
#959233: SSH: add inline script support

### PR DESCRIPTION
**Pull request by lkillgore**

### CODE REVIEW

File: Tasks/SSH/ssh.ts [hunk 1]
```diff
@@ -55,6 +55,19 @@ async function run() {
 
         if(runOptions === 'commands') {
             commands = tl.getDelimitedInput('commands', '\n', true);
+        } else if (runOptions === 'inline') {
+            var inlineScript: string = tl.getInput('inline', true);
+            const scriptHeader:string = '#!';
+            if (inlineScript && !inlineScript.startsWith(scriptHeader)) {
+                inlineScript = '#!/bin/bash' + os.EOL + inlineScript;
+            }
+            scriptFile = path.join(os.tmpdir(), 'sshscript_' + new Date().getTime()); // default name
```
Comment (DavidStaheli) [inline Tasks/SSH/ssh.ts]: Will getTime() produce ":" characters that aren't valid in Windows file names?

File: Tasks/SSH/ssh.ts [hunk 2]
```diff
@@ -55,6 +55,19 @@ async function run() {
 
         if(runOptions === 'commands') {
             commands = tl.getDelimitedInput('commands', '\n', true);
+        } else if (runOptions === 'inline') {
+            var inlineScript: string = tl.getInput('inline', true);
+            const scriptHeader:string = '#!';
+            if (inlineScript && !inlineScript.startsWith(scriptHeader)) {
+                inlineScript = '#!/bin/bash' + os.EOL + inlineScript;
```
Comment (DavidStaheli) [inline Tasks/SSH/ssh.ts]: Can you output an informational message that says we're adding this to their script?

File: Tasks/SSH/ssh.ts (hunk 1 above)
Comment (lkillgore) [inline Tasks/SSH/ssh.ts]: getTime() returns millis, so we're good.

File: Tasks/SSH/ssh.ts (hunk 2 above)
Comment (lkillgore) [inline Tasks/SSH/ssh.ts]: Yes!

**Pull request merged**